session-10/src/components/Header/Header.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header-light {
+  background-color: #272e35;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-dark {
+  background-color: #ffff;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+input,
```

Comment:
Keep your body, *, and html-tag CSS selectors on top, keeping in mind CSS specificity.